data/json/item_groups.json

```diff
@@ -8386,7 +8393,6 @@
     ]
   },
   {
-    "id": "museum_primitive",
```

Comment:
Why are you removing `id` of this item group?

Reply:
Thanks, whoops. Moved all museum stuff to it's own json as part of overall json organization. This was just a screwup - didn't get the whole category moved. Good catch.